config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  resources :doctors, only: [:new, :create, :index, :show, :destroy]
+  resources :users, only: [:new, :create, :index] do
+    resources :bookings, only: [:new, :index, :show, :create, :destroy]
+  end
```

Comment:
- Please, it looks like the documentation isn't all the routes, I kindly invite you to add all of them to the docs. 🐒 
<img width="1584" alt="image" src="https://user-images.githubusercontent.com/20563252/207433599-fd2e85fe-b990-44dc-b637-e855b4ca768c.png">

Reply:
Done, Thanks